_posts/events/2016-11-11-iot-in-the-city-collider.md

```diff
@@ -8,7 +8,7 @@ speakers:
     - Simon Chapple
     - Ritchie Somerville
     - Eoghan Howard
-venue: design-informatics
+venue: "Design Informatics"
```

Comment:
I'm not sure why you've done this. The value `design-informatics` is a key for the venues list in the data directory. Cf https://blog.sorryapp.com/blogging-with-jekyll/2014/02/06/adding-authors-to-your-jekyll-site.html